code/networking/flask-server.py

```diff
@@ -47,13 +47,15 @@ def __init__(self):
 		self.strobe = strobe.StrobeLights()
 		self.column = columns.ColumnLights()
 		self.blinker = blink.BlinkLights()
+		self.crazy = crazy.CrazyOldColumnLights()
 
 		self.functionMap = {
 			"twinkle": self.twinkler.run,
 			"stripes": self.striper.run,
 			"strobe": self.strobe.run,
 			"columns": self.column.run,
 			"blink": self.blinker.run,
+			"crazy": self.crazy.run,

```

Comment:
oh I see you just call them cazy here. As long as the `OldColumn` part isn't everywhere.